runelite-client/src/main/java/net/runelite/client/plugins/timetracking/TabContentPanel.java

```diff
@@ -24,11 +24,14 @@
  */
 package net.runelite.client.plugins.timetracking;
 
+import net.runelite.client.RuneLite;
+import net.runelite.client.util.StackFormatter;
```

Comment:
Fix your imports

Reply:
Moved to bottom of imports which is I assume what you mean.